### PR TITLE
MOW-125: show non-public Notes sent to followers

### PR DESCRIPTION
**Pull request by xtuc**

### CODE REVIEW

File: backend/src/mastodon/timeline.ts [hunk 1]
```diff
@@ -10,10 +9,44 @@ export async function pregenerateTimelines(domain: string, db: D1Database, cache
...
+        `
+		)
+		.bind(actor.id.toString())
+		.all<ActorFollowingResult>()
+
+	let followingIds: string[] = []
+	let followingFollowersURLs: string[] = []
+
+	if (following) {
+		followingIds = following.map((row: ActorFollowingResult) => row.id)
```
[the remark this answers is not in the document]
Comment (dario-piotrowicz) [reply backend/src/mastodon/timeline.ts:36]: here and on the next line as well

File: backend/src/mastodon/timeline.ts [hunk 2]
```diff
@@ -10,10 +9,44 @@ export async function pregenerateTimelines(domain: string, db: D1Database, cache
...
+		followingFollowersURLs = following.map((row: ActorFollowingResult) => {
+			if (row.actorFollowersURL) {
+				return row.actorFollowersURL
+			} else {
+				// We don't have the Actor's followers URL stored, we'll guess
+				// one.
+				return row.id + '/followers'
+			}
+		})
```
Comment (dario-piotrowicz) [inline backend/src/mastodon/timeline.ts:45]:
```suggestion
		followingFollowersURLs = following.map(({actorFollowersURL}: ActorFollowingResult) => 
			actorFollowersURL ?? `${row.id}/followers`
		)
```
also as I mentioned above the type may not be needed?

**Review by dario-piotrowicz — APPROVED** [review]

File: backend/src/mastodon/timeline.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (xtuc) [reply backend/src/mastodon/timeline.ts]: You are right, it can infer it. I removed it

File: backend/src/mastodon/timeline.ts (hunk 2 above)
Comment (xtuc) [reply backend/src/mastodon/timeline.ts]: Removed

**Pull request merged**